projects/Mallard/src/components/styled-text.tsx

```diff
@@ -1,30 +1,64 @@
 import React from 'react'
 import { Text, View } from 'react-native'
-import { color } from '../theme/color'
+import { useAppAppearance } from '../theme/appearance'
 import { metrics } from '../theme/spacing'
+import { color } from '../theme/color'
 
-export class HeadlineText extends React.Component<{ style?: any }> {
-    render() {
-        return (
-            <Text
-                {...this.props}
-                style={{
-                    fontFamily: 'GHGuardianHeadline-Bold',
+export const HeadlineText = ({
+    style,
+    ...props
+}: {
+    children: string
+    style?: {}
+}) => {
+    return (
+        <Text
+            {...props}
+            style={[
+                {
+                    fontFamily: 'GHGuardianHeadline-Medium',
```

Comment:
Would be awesome to extract these font families into `theme/typography` at some point

Reply:
since all font styles are centralized in this file I don't wanna take type unification too seriously until we get final designs tbh